calendar2report.js

```diff
@@ -72,21 +75,20 @@ function groupByDate(iterator, current) {
   return iterator;
 }
 
-
 async function main() {
   let timeMin = moment('2020-01-01').toISOString();
   let timeMax = moment().toISOString();
 
   let events = await calendar.retrieveEvents(timeMin, timeMax);
 
-  events = events.map(asCompactedEvent)
-    .filter(event => event.attended) // only when I attended
-    .filter(event => event.started) // only if duration is not 24h
-    .filter(event => !event.outofoffice) // skip if I'm supposed to be out -- TODO others like gym, lunch, ..
-    .filter(event => event.title !== 'lunch')
+  events = events
+    .map(asCompactedEvent)
+    .filter((event) => event.attended) // only when I attended
+    .filter((event) => event.started) // only if duration is not 24h
+    .filter((event) => !event.outofoffice) // skip if I'm supposed to be out -- TODO others like gym, lunch, ..
+    .filter((event) => event.title !== 'lunch')
```

Comment:
```.filter((event) => !event.outofoffice)```
already filters out events like gym, lunch, etc ... because those will have a type of outOfOffice